packages/engine-backend/services/sync-service.ts

```diff
@@ -19,6 +19,8 @@ import {
 } from '@openint/cdk'
 import type {z} from '@openint/util'
 import {rxjs} from '@openint/util'
+// Amadeo Q: how do I make the atsLink part of the openint/cdk? is there some sort of release process?
```

Comment:
it should not be part of cdk. it's a dependecy graph issue that I should sort out. `cdk` stands for connector development kit and it should be depended on BY things like connectors and unified APIs, not the other way around. Just a code smell that we need to fix unfortunately

Reply:
i see, got confused by banking link being imported here and tried to reproduce it but its not part of the cdk package, hence the question

```
import {
  bankingLink,
  logLink,
  makeId,
  prefixConnectorNameLink,
  sync,
} from '@openint/cdk'
```